### PR TITLE
Add resilience doc

### PR DESCRIPTION
**Pull request by rvowles**
Docs updates - resilience, backup feature flags

### CODE REVIEW

File: docs/modules/ROOT/pages/resilience.adoc [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+There are several approaches, and you can mix and match.
+
+==== assume a state for all of your features as part of your code development
+
+This would mean for example that you assume or code for all features being false or deliberately choosing a state for them. In FeatureHub SDK's, feature flags for instance can have one of three states: non-existent, false or true, where non-existent becomes automatically false unless you code for it.
+
+==== provide backup source state for your feature flags
+
+This usually entails that you either a file representing your feature flags that is available somewhere. It could be as part of the base docker image, loaded as a configmap or secret, available in an S3 or Cloud Storage bucket or otherwise. The SDKs have a method of loading external state into the SDK.
```
Comment (IrinaSouth) [inline docs/modules/ROOT/pages/resilience.adoc:42]: Didn't get this first sentence.. Should it just say "You can store feature flags states in a file?"

File: docs/modules/ROOT/pages/resilience.adoc [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+All SDK's also support _interceptors_ - which are ways to override how features are determined, so they can be used to provide
```
Comment (IrinaSouth) [inline docs/modules/ROOT/pages/resilience.adoc:44]: If there is a doc about Interceptors, would be great to link to it

File: docs/modules/ROOT/pages/resilience.adoc [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+state. One example of an interceptor for example is where OpenTelemetry is used and the Baggage is examined for state pre-set
+from upstream sources. 
+
+==== provide no backup source
```
Comment (IrinaSouth) [inline docs/modules/ROOT/pages/resilience.adoc:48]: Do we really need to include this? Isn't it just default setup/behavior?

File: docs/modules/ROOT/pages/resilience.adoc [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+- FeatureHub is capable of directly integrating with Fastly's worldwide distributed cache, and this comes out of the box, even in the Open Source version! mailto:info@featurehub.io[Please contact us] for details and we can provide you with configuration.
+
+In SaaS, we use Fastly's worldwide distributed cache making the chances of FeatureHub not being available extremely low.
+
+== Resilient SDKs
+
+Resiliency in SDKs face two main issues:
+
+- If FeatureHub is unavailable, how do you get your initial state and,
+- If FeatureHub becomes available _after_ startup, how do you get your updated state
```
Comment (IrinaSouth) [inline docs/modules/ROOT/pages/resilience.adoc:27]: I do not get it, if it is available, then the features should be available, isn't that the case? What am I missing?

Comment (rvowles) [reply docs/modules/ROOT/pages/resilience.adoc:27]: Updated this wording to: "If FeatureHub becomes available _after_ startup (i.e. it was not available at the time of startup and you have gone to a backing store to get initial state), how do you get your updated state"

File: docs/modules/ROOT/pages/resilience.adoc (hunk 1 above)
Comment (rvowles) [reply docs/modules/ROOT/pages/resilience.adoc:42]: Ok, I have updated this text to see if I can make it more obvious. I don't want to say "file" as it is really just a JSON blob.

File: docs/modules/ROOT/pages/resilience.adoc (hunk 2 above)
Comment (rvowles) [reply docs/modules/ROOT/pages/resilience.adoc:44]: I think I added the link, will gen

File: docs/modules/ROOT/pages/resilience.adoc (hunk 3 above)
Comment (rvowles) [reply docs/modules/ROOT/pages/resilience.adoc:48]: ok, cut - good call, i was dubious

File: docs/modules/ROOT/pages/resilience.adoc [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+==== assume a state for all of your features as part of your code development
+
+This would mean for example that you assume or code for all features being false or deliberately choosing a state for them. In FeatureHub SDK's, feature flags for instance can have one of three states: non-existent, false or true, where non-existent becomes automatically false unless you code for it.
+
+==== provide backup source state for your feature flags
+
+This usually entails that you have a JSON blob representing your feature flags that is available somewhere. It could be as part of the base docker image, loaded as a configmap or secret in Kubernetes, available in an S3 or Cloud Storage bucket, in a key storage (e.g. Redis, Memcache or DynamoDB) or otherwise. The SDKs have a method of loading external state into the SDK.
+
+All SDK's also support xref:capabilities-explained.adoc#_feature_interceptors[_interceptors_] - which are ways to override how features are determined, so they can be used to provide
+state. One example of an interceptor for example is where OpenTelemetry is used and the Baggage is examined for state pre-set
```
Comment (IrinaSouth) [inline docs/modules/ROOT/pages/resilience.adoc:45]:
```suggestion
state. One example of an interceptor  is where OpenTelemetry is used and the Baggage is examined for state pre-set
```

**Review by IrinaSouth — APPROVED** [review]

**Pull request merged**